app/my-bets/page.tsx

```diff
@@ -23,6 +29,19 @@ export default function MyBetsPage() {
     enabled: !!address,
   });
 
+  const userPositions = data?.userPositions ?? [];
+  const emptyData = data && data.userPositions.length == 0;
+
+  const filterActiveBets = (userPositions: UserPosition[]) =>
+    userPositions?.filter(
+      position => position.position.conditions[0].resolved === false
+    );
+
+  const filterClompleteBets = (userPositions: UserPosition[]) =>
+    userPositions?.filter(position => position.position.conditions[0].resolved);
```

Comment:
No need for a function here, we can filter directly with `userPositions`.
```suggestion
  const filterActiveBets = userPositions?.filter(
      position => position.position.conditions[0].resolved === false
    );

  const filterClompleteBets = userPositions?.filter(position => position.position.conditions[0].resolved);
```

Reply:
true, fixed.